### PR TITLE
Fixed the case on "Apollo" and added a 'sudo' note

### PR DESCRIPTION
**Pull request by andrewharmellaw**
Ludicrously small change.

### CODE REVIEW

File: docs/getting-started-guides/vagrant.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Running Apollo with Vagrant (and Virtualbox) is an easy way to run/test/develop
 
 1. The latest version of vagrant installed (>= 1.7.2) [http://www.vagrantup.com/downloads.html](http://www.vagrantup.com/downloads.html) to get started
 2. Install vagrant-hosts plugin ```vagrant plugin install vagrant-hosts```
-3. The latest version of Virtualbox from [https://www.virtualbox.org/wiki/Downloads](https://www.virtualbox.org/wiki/Downloads)
+3. The latest version of Virtualbox from [https://www.virtualbox.org/wiki/Downloads](https://www.virtualbox.org/wikiDownloads)
```
Comment (tayzlor) [inline docs/getting-started-guides/vagrant.md]: Did you mean to change the URL here?

File: docs/getting-started-guides/vagrant.md [hunk 2]
```diff
@@ -14,10 +14,12 @@ Running Apollo with Vagrant (and Virtualbox) is an easy way to run/test/develop
 #### Download Apollo
 
 ##### Install from source at head
-1. ```git clone https://github.com/Capgemini/apollo.git```
-2. ```cd apollo```
+1. ```git clone https://github.com/Capgemini/Apollo.git```
+2. ```cd Apollo```
 3. ```pip install -r requirements.txt```
 
+_Note: you need to run the pip command with root priviledges (e.g. sudo)_
```
Comment (tayzlor) [inline docs/getting-started-guides/vagrant.md]: I guess this depends on your OS and what the perms are for the folder, might be worth putting in a ``` you *may*``` -

File: docs/getting-started-guides/vagrant.md (hunk 1 above)
Comment (andrewharmellaw) [inline docs/getting-started-guides/vagrant.md]: Nope, thats me flailing around on the keyboard. I'll put that back.

File: docs/getting-started-guides/vagrant.md (hunk 2 above)
Comment (andrewharmellaw) [inline docs/getting-started-guides/vagrant.md]: wilko.  I'll update.

**Comment by tayzlor** [comment]
Thanks!

**Comment by andrewharmellaw** [comment]
NP. More coming... ;-)

**Pull request merged**